bigframes/ml/preprocessing.py

```diff
@@ -267,6 +271,113 @@ def transform(self, X: Union[bpd.DataFrame, bpd.Series]) -> bpd.DataFrame:
         )
 
 
+class KBinsDiscretizer(
+    base.Transformer,
+    third_party.bigframes_vendored.sklearn.preprocessing._discretization.KBinsDiscretizer,
+):
+    __doc__ = (
+        third_party.bigframes_vendored.sklearn.preprocessing._discretization.KBinsDiscretizer.__doc__
+    )
+
+    def __init__(
```

Comment:
If we don't support sklearn's default only for now, make sure to document explicitly. Also still make 'quantile' as default, but throw an error for it. 

I.e. user needs to explicitly call strategy = 'uniform'.

Reply:
Done adding strategy as the input param.